### PR TITLE
Windows test fix

### PR DESCRIPTION
**Pull request by targos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Do not use this in production!
 
 ```bash
 docker pull couchdb
-docker create -p 5984:5984 --name couchdb -v /your/local/path/of/choice:/opt/couchdb/data couchdb
```
Comment (stropitek) [inline README.md]: This doesn't work on windows?

Comment (targos) [reply README.md]: It probably works, but I think it's generally useless to have this data in the local filesystem (see commit message)

Comment (stropitek) [reply README.md]: If you don't do this you'll have to redo the couchdb setup every time you launch the docker instance.

Comment (targos) [reply README.md]: No, only when you delete and recreate the container. That's why we give it a name: to reuse its data storage.

**Pull request merged**